doc/pr2_tutorials/planning/src/move_group_interface_tutorial.cpp

```diff
@@ -43,47 +43,66 @@
 #include <moveit_msgs/AttachedCollisionObject.h>
 #include <moveit_msgs/CollisionObject.h>
 
+#include <moveit_visual_tools/moveit_visual_tools.h>
+
 int main(int argc, char **argv)
 {
   ros::init(argc, argv, "move_group_interface_tutorial");
   ros::NodeHandle node_handle;
   ros::AsyncSpinner spinner(1);
   spinner.start();
 
-  /* This sleep is ONLY to allow Rviz to come up */
-  sleep(20.0);
-
   // BEGIN_TUTORIAL
   //
   // Setup
   // ^^^^^
-
+  //
   // MoveIt! operates on sets of joints called "planning groups" and stores them in an object called
   // the `JointModelGroup`. Throughout MoveIt! the terms "planning group" and "joint model group"
   // are used interchangably.
   static const std::string PLANNING_GROUP = "right_arm";
 
-  // The :move_group_interface:`MoveGroup` class can be easily
+  // The :move_group_interface:`MoveGroupInterface` class can be easily
   // setup using just the name of the planning group you would like to control and plan for.
   moveit::planning_interface::MoveGroupInterface move_group(PLANNING_GROUP);
 
+  // We will use the :planning_scene_interface:`PlanningSceneInterface`
+  // class to add and remove collision objects in our "virtual world" scene
+  moveit::planning_interface::PlanningSceneInterface planning_scene_interface;
+
   // Raw pointers are frequently used to refer to the planning group for improved performance.
   const robot_state::JointModelGroup *joint_model_group =
-      move_group.getCurrentState()->getJointModelGroup(PLANNING_GROUP);
+    move_group.getCurrentState()->getJointModelGroup(PLANNING_GROUP);
 
-  // (Optional) Create a publisher for visualizing plans in Rviz.
-  ros::Publisher display_publisher =
-      node_handle.advertise<moveit_msgs::DisplayTrajectory>("/move_group/display_planned_path", 1, true);
-  moveit_msgs::DisplayTrajectory display_trajectory;
+  // Visualization
+  // ^^^^^^^^^^^^^
+  //
+  // The package MoveItVisualTools provides many capabilties for visualizing objects, robots,
+  // and trajectories in Rviz as well as debugging tools such as step-by-step introspection of a script
+  namespace rvt = rviz_visual_tools;
+  moveit_visual_tools::MoveItVisualTools visual_tools("odom_combined");
+  visual_tools.deleteAllMarkers();
+
+  // Remove control is an introspection tool that allows users to step through Rviz via buttons and keyboard
+  // shortcuts in Rviz
```

Comment:
Thanks, revised to "Remote control is an introspection tool that allows users to step through a high level script via buttons and keyboard shortcuts in Rviz"